src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -57,6 +60,13 @@ public ExcessHet(final File pedigreeFile){
         super(pedigreeFile);
     }
 
+    @Override
+    void validateArguments(Collection<String> founderIds, File pedigreeFile) {
+        if ((founderIds == null || founderIds.isEmpty()) && pedigreeFile == null) {
+            warning.warn("ExcessHet annotation will not be calculated, no 'founder-id' or 'pedigree' arguments provided");
```

Comment:
These error message strings should use the command line argument name constants (i.e. `StandardArgumentDefinitions.PEDIGREE_FILE_LONG_NAME`).

Reply:
good call